packages/datadog_flutter_plugin/android/src/main/kotlin/com/datadoghq/flutter/DatadogRumPlugin.kt

```diff
@@ -207,6 +209,18 @@ class DatadogRumPlugin(
                         result.missingParameter(call.method)
                     }
                 }
+                "reportLongTask" -> {
+                    val at = call.argument<Long>(PARAM_AT)
+                    val duration = call.argument<Int>(PARAM_DURATION)
+                    if (at != null && duration != null) {
+                        // Duration is in ms, convert to ns
+                        val durationNs = duration.toLong() * 1000
```

Comment:
1 millisecond is equal to `1 000 000` nanosecond.
To avoid hardcoding conversion rates, you can use the `TimeUnit` helper class:

```
val durationNs = TimeUnit.MILLISECONDS.toNanos(duration.toLong())
```